### PR TITLE
Add Safe events service

### LINKED ISSUE #100: Add safe-events-service to infrastructure
# Description
This issue is created to add to safe-infrastructure the new safe-events https://github.com/safe-global/safe-events-service.

# Tasks
 - [x] Include safe events service in docker compose
 - [x] Update README
 - [x] #113

### LINKED ISSUE #113: Update diagram of infrastructure

### PR DESCRIPTION
**Pull request by moisses89**
Closes #100 
Closes #113 
- Update Readme and add diagrams section to store raw diagrams and picture diagrams. 
- Update local instructions to use safe-events-service to send webhooks instead transaction service. 
- Add safe-events-service and dependencies to docker-compose file. 
- Add default environment variables for safe-events-service.

### CODE REVIEW

**Comment by moisses89** [comment]
@Uxio0 this PR is ready for review, but we shouldn't merge it until get the version 0.5.0 of safe-events-service.

**Comment by Uxio0** [comment]
Depends on https://github.com/safe-global/safe-events-service/pull/96

File: .env.sample [hunk 1]
```diff
@@ -5,5 +5,6 @@ CFG_VERSION=latest
 CGW_VERSION=latest
 TXS_VERSION=latest
 UI_VERSION=latest
+EVS_VERSION=latest
```
Comment (Uxio0) [inline .env.sample:8]: I would prefer to use EVENTS_VERSION. For TXS and CGW it makes sense, but I think EVS is really short and a little difficult to understand

File: docker-compose.yml [hunk 2]
```diff
@@ -32,6 +32,7 @@ services:
       - txs-web
       - cfg-web
       - cgw-web
+      - evs-web
```
Comment (Uxio0) [inline docker-compose.yml:35]: Same, I would prefer events-web, events-db...

File: docker/nginx/nginx.conf [hunk 3]
```diff
@@ -51,6 +51,15 @@ http {
       keepalive 32;
     }
 
+    upstream evs_app_server {
```
Comment (Uxio0) [inline docker/nginx/nginx.conf:54]: Same as before, evs -> events

File: docs/running_locally.md [hunk 4]
```diff
@@ -83,14 +83,17 @@ AUTH_TOKEN=some_random_token
...
 `AUTH_TOKEN` and `CGW_FLUSH_TOKEN` must be the same.
 
-For the Transactions service, follow these steps:
+For the Events service, follow these steps:
 
-- Access the admin panel at `http://localhost:8000/txs/admin`
-- click the `Add` link for `Web hooks`
-- Ignore the `Address` field
-- Set the `Url` field to `http://nginx:8000/cgw/v1/chains/{chainId}/hooks/events` and replace `{chainId}` with the corresponding chainId
+- Access the admin panel at `http://localhost:8000/evs/admin`
```
Comment (Uxio0) [inline docs/running_locally.md:88]: Same, events

File: docs/running_locally.md [hunk 5]
```diff
@@ -83,14 +83,17 @@ AUTH_TOKEN=some_random_token
 
 `AUTH_TOKEN` and `CGW_FLUSH_TOKEN` must be the same.
 
-For the Transactions service, follow these steps:
+For the Events service, follow these steps:
```
Comment (Uxio0) [inline docs/running_locally.md:86]: For the future, we should make this automatic, adding a management endpoint on the events service. But it's also good that users know what's going on

**Review by Uxio0 — CHANGES_REQUESTED** [review]
Great work, just some small naming requests

File: docs/running_locally.md (hunk 5 above)
Comment (moisses89) [reply docs/running_locally.md:86]: Agree, at least for the client-gateway.

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**